L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -118,5 +119,13 @@ def customiseSimL1EmulatorForPostLS1_HI(process):
     if hasattr(process,'caloConfig'):
         process.caloConfig.fwVersionLayer2 = cms.uint32(1)
     # move to the heavy ions draft L1 menu once the HLT has been updated accordingly
-    process = L1Menu_CollisionsHeavyIons2015_v0(process)
+    process = L1Menu_CollisionsHeavyIons2015_v1(process)
```

Comment:
Indeed, this was mistakenly brought into the unpacker update... will be fixed in #10087.